addons/base_vehicles/CfgVehicles_Polaris.hpp

```diff
@@ -0,0 +1,44 @@
+class B_LSV_01_unarmed_F;
+class GCLASS(Polaris) : B_LSV_01_unarmed_F {
+    displayName = "Polaris DAGOR";
+    faction = QGCLASS(base_vehicles);
+    SCOPE_DLC_VEHICLE;
+    EMPTY_INVENTORY;
+    crew = "Civilian";
+    typicalCargo[] = {"Soldier"};
+    ace_refuel_fuelCapacity = 190;
```

Comment:
Where are you getting the numbers from? The Polaris DAGOR should be closer to 95 litres

Reply:
I've been adding them to everything in this repo